ui/src/page/game.rs

```diff
@@ -100,6 +102,11 @@ pub fn view(model: &Model) -> Vec<Node<Msg>> {
 
         grass_tile_rows.push(row(grass_tile_cells));
     }
+    let timestamp_str = session.get_current_time().to_string();
+
+    let timestamp_row = row::single(text(timestamp_str.as_str()));
+
+    grass_tile_rows.push(timestamp_row);
```

Comment:
RIght now this is all that we are doing with the timestamp, rendering it as text at the bottom of the page.

Reply:
Not sure  why we need that but im sure its works :)